src/Compilers/CSharp/Portable/CodeGen/EmitConversion.cs

```diff
@@ -74,7 +74,7 @@ private void EmitReadOnlySpanFromArrayExpression(BoundReadOnlySpanFromArray expr
                          this._module.Compilation.IsReadOnlySpanType(typeTo),
                          "only special kinds of conversions involving ReadOnlySpan may be handled in emit");
 
-            if (!TryEmitReadonlySpanAsBlobWrapper(typeTo, operand, used, inPlace: false))
+            if (!TryEmitReadonlySpanAsBlobWrapper(typeTo, operand, used, inPlaceTarget: null, out _))
```

Comment:
>out _ [](http://example.com/codeflow?start=94&length=5)

nit: consider naming discarded `out` arguments to `TryEmitReadonlySpanAsBlobWrapper` (here and elsewhere)